tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ES2018",
```

Comment:
Why?

Reply:
We updated one of the regex parsers (forgot which dependency but that dependency require this)

Reply:
Yes but why ES2018? And not ES2021, [fully supported](https://node.green/#ES2021) by Node.js v20?

Reply:
Oh i went with the minimum change actually in fear of anything breaking but ill change it

Reply:
@fallenbagel you forgot to update this

Reply:
Oh yes